pvdeg/weather.py

```diff
@@ -6,7 +6,8 @@
 import os
 import glob
 import pandas as pd
-from rex import NSRDBX, Outputs
+from rex import NSRDBX as f
```

Comment:
this must be a typo, NSRDBX should not be renamed

Reply:
I fixed this back to how it was. I had misunderstood how some functions later in the code were using f.*** to reference code from rex.

Reply:
great, I don't see any error anymore with that.